### PR TITLE
Fix Update method on EtClient

### PR DESCRIPTION
**Pull request by rlweb**
The update method is using the wrong params and names

### CODE REVIEW

File: src/druid628/exactTarget/EtClient.php [hunk 1]
```diff
@@ -28,7 +28,7 @@ class EtClient extends EtBaseClass
     const SOAPWSDL = "http://exacttarget.com/wsdl/partnerAPI";
 
     /** @var \druid628\exactTarget\EtSoapClient $client */
-    protected $client;
+    public $client;
```
Comment (druid628) [inline src/druid628/exactTarget/EtClient.php]: Why was this changed to public?

Comment (rlweb) [inline src/druid628/exactTarget/EtClient.php]: We're using it internally to get the soap request and response for debugging in certain environments, although it can be changed back

Comment (druid628) [inline src/druid628/exactTarget/EtClient.php]: Gotcha. If you'll change that back to protected I'll go ahead and merge this.
Thanks so much for this fix.

Comment (rlweb) [inline src/druid628/exactTarget/EtClient.php]: Updated!
